tasks/set_facts.yml

```diff
@@ -7,6 +7,24 @@
     difference(ansible_facts.keys() | list) | length > 0
   no_log: true
 
+- name: Ensure correct package manager for ostree systems
```

Comment:
I can not understand what this task is intended for, maybe add a comment explaining in detail about the purpose of this task ?

Reply:
Well, the Ansible guidelines state that you should use the `name:` of a task to adequately describe what the task does.  So how about this:

`- name: Ensure the use of the ostree package manager on ostree systems`

If that is still not clear enough, I guess I would have to ask what is unclear.

Reply:
I could add a section to the README-ostree.md which describes in detail things like
- why isn't this functionality in ansible-core?
- what determines if a system is an ostree system?
- why does an ostree system require an "external" package manager for use by the `package` and `package_facts` modules?
- other faqs?

Reply:
Is this good now, as-is?  Further changes required?

Reply:
I have doubts about why an ostree system require an "external" package manager for use. Now I understand that none of the built-in package managers in Ansible can be used on ostree system, because that we can not install packages in an ostree system. Thanks for explaining it.